linux/jdk/debian/src/main/packaging/temurin/19/debian/rules

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/make -f
+
+pkg_name = temurin-19-jdk
+priority = 1811
+jvm_tools = jar jarsigner java javac javadoc javap jcmd jconsole jdb jdeprscan jdeps jfr jhsdb jimage jinfo jlink jmap jmod jpackage jps jrunscript jshell jstack jstat jstatd jwebserver keytool rmiregistry serialver jexec jspawnhelper
+amd64_tarball_url = https://github.com/adoptium/temurin19-binaries/releases/download/jdk-19%2B36/OpenJDK18U-jdk_x64_linux_hotspot_19_36.tar.gz
```

Comment:
these links need to be updated  OpenJDK19U-

Reply:
updated